pkg/k8s/watchers/subscriber/service.go

```diff
@@ -57,7 +57,7 @@ func (l *ServiceList) NotifyUpdate(oldSvc, newSvc *slim_corev1.Service) {
 	}
 }
 
-// NotifyDelete notifies all the subscribers of an update event to a service.
+// NotifyDelete notifies all the subscribers of a delete event to a service.
```

Comment:
Should this and other comment fixed go in a separate commit?

Reply:
If there is code change surely Yes. Since its minor comment change, i think it's okay.